### PR TITLE
Add OpenApi gen service tests

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/CodeGenerator.java [hunk 1]
```diff
@@ -147,15 +148,17 @@ public void generateClient(String executionPath, String definitionPath, String o
...
      *                       {@code definitionPath} will be used as the default destination path
      * @throws IOException               when file operations fail
      * @throws BallerinaOpenApiException when code generator fails
      */
-    public void generateService(String executionPath, String definitionPath, String serviceName, String outPath)
+    public void generateService(String executionPath, String definitionPath,
+                                String reldefinitionPath, String serviceName, String outPath)
             throws IOException, BallerinaOpenApiException {
-        generate(GenType.GEN_SERVICE, executionPath, definitionPath, serviceName, outPath);
+        generate(GenType.GEN_SERVICE, executionPath, definitionPath, reldefinitionPath , serviceName, outPath);
```
Comment (NipunaMarcus) [inline misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/CodeGenerator.java]: there is a formatting issue extra space between `reldefinitionPath ,`

File: misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/cmd/OpenApiGenServiceCmd.java [hunk 2]
```diff
@@ -182,11 +190,21 @@ public void execute() {
...
         generator.setSrcPackage(moduleArgs.get(0));
 
+        //Set relative path for contract path which will be printed on the generated service bal file
+        Path absPath = Paths.get(resourcePath.toString());
+        Path basePath = Paths.get(output);
+        Path pathRelative = basePath.relativize(absPath);
+
         try {
-            generator.generateService(executionPath, resourcePath.toString(), moduleArgs.get(1), output);
+            generator.generateService(executionPath, resourcePath.toString() , pathRelative.toString(),
```
Comment (NipunaMarcus) [inline misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/cmd/OpenApiGenServiceCmd.java]: formatting issue additional space between comma and parameter

Comment (DimalChandrasiri) [reply misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/cmd/OpenApiGenServiceCmd.java]: Fixed with commit c1f65eb

File: misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/CodeGenerator.java (hunk 1 above)
Comment (DimalChandrasiri) [reply misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/CodeGenerator.java]: Fixed with commit c1f65eb

**Pull request merged**